node-src/ui/messages/errors/unparseableConfigurationFile.ts

```diff
@@ -3,9 +3,11 @@ import { dedent } from 'ts-dedent';
 
 import { error } from '../../components/icons';
 
-export const unparseableConfigurationFile = (configFile: string, err: Error) =>
-  dedent(chalk`
-    ${error} Configuration file {bold ${configFile}} could not be parsed, is it valid JSON?
+export const unparseableConfigurationFile = (configFile: string, err: Error) => {
+  const language = configFile.endsWith('.jsonc') ? 'JSON5' : 'JSON';
```

Comment:
Should we also use `JSON5` for `.json5` files?

Reply:
Ah, yeah, I can't believe I missed that lol.

Reply:
Done!